CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 2.19
+### Added
+- Option to add/edit whitelist or ignore patterns to yamllint.
```

Comment:
I thought this was already possible with e.g this in your grumphp file

```
grumphp:
  tasks:
    yamllint:
      whitelist_patterns:
        - ^bamboo-specs\/(.*)
```

Reply:
@rutgerrademaker But do you want to have this exclude in every project setup?

Reply:
yes the same as composer.strict:false :(

Reply:
Only when bamboo yaml does not validate
In my case it failed on multi-docs (---) and includes (!)
Ideally I could just ignore those, but did not find a solution for that yet